kong/plugins/oauth2/migrations/cassandra.lua

```diff
@@ -74,37 +75,41 @@ return {
   },
   {
     name = "2016-04-14-283949_serialize_redirect_uri",
-    up = function(_, _, factory)
-      local json = require "cjson"
-      local apps, err = factory.oauth2_credentials.db:find_all('oauth2_credentials', nil, nil);
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
```

Comment:
This SELECT construct will not return all elements. For row-by-row updates in Cassandra migrations, you need to use the `iterate` construct as shown here:
https://github.com/Kong/kong/blob/next/kong/dao/migrations/cassandra.lua#L586-L603

Likewise in the migrations below.

Reply:
OMG I always forget that. Thanks.